code/modules/atmospherics/gasmixtures/reactions.dm

```diff
@@ -133,6 +137,59 @@ nobliumformation = 1001
 		return REACTING
 	return NO_REACTION
 
+/datum/gas_reaction/condensation
+	priority = 0
+	name = "Condensation"
+	id = "condense"
+	exclude = TRUE
+	var/datum/reagent/condensing_reagent
+
+/datum/gas_reaction/condensation/New(datum/reagent/condensed_reagent)
+	. = ..()
+	if(!istype(condensed_reagent))
+		return
+	min_requirements = list(
+		"MAX_TEMP" = initial(condensed_reagent.condensating_point)
+	)
+	min_requirements[condensed_reagent.get_gas()] = MOLES_GAS_VISIBLE
+	name = "[condensed_reagent.name] condensation"
+	id = "[condensed_reagent.type] condensation"
+	condensing_reagent = GLOB.chemical_reagents_list[condensed_reagent.type]
+	exclude = FALSE
+
+/datum/gas_reaction/condensation/react(datum/gas_mixture/air, datum/holder)
+	. = NO_REACTION
+	if(prob(95)) //these reactions fire way to fucking much future me make this not shit and give reagents reaction probs - Borbop
+		return
```

Comment:
I'm not liking that this method. It makes it still fire the proc, wasting processing time.
Chem gasses are going to need to become a subsystem if they need to run at a certain speed.

Reply:
Own SS most likely wouldn't work, as it would than stall reactions to keep up with it, unless we moved chem gases to its own ss but that idea was shot down with pollutions